core/modules/layout/layout.admin.inc

```diff
@@ -271,12 +271,21 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     }
   }
 
+  $items = array(
+    t('Content: &nbsp; <code>node/%</code>'),
+    t('User accounts: &nbsp; <code>user/%</code>'),
+    t('Taxonomy terms: &nbsp; <code>taxonomy/term/%</code>'),
+    t('Comments: &nbsp; <code>comment/%</code>'),
+  );
+  $description_list = theme('item_list', array('items' => $items));
+
   $form['path'] = array(
     '#title' => t('Path'),
+    '#field_prefix' => url('', array('absolute' => TRUE)),
     '#type' => 'textfield',
     '#default_value' => $layout->getPath(),
     '#required' => TRUE,
-    '#description' => t('Use the "%" (percent) symbol to indicate a wildcard in the path. i.e. "node/%" or "node/%/edit".'),
+    '#description' => '<p>' . t('Use the percent symbol ("%") to indicate a placeholder in the path. Some common examples include:') . '</p>' . $description_list,
```

Comment:
No need for `<p>` tags here either. This works just fine:

```suggestion
    '#description' => t('Use the percent symbol ("%") to indicate a placeholder in the path. Some common examples include:') . ' ' . $description_list,
```

Reply:
Again, I found this hard to read when all squished together :) But I'm going to merge this because it's done and I like the code better this way. DX over UX?